### PR TITLE
EDIT & DELETE button's position change

### PR DESCRIPTION
**Pull request by biswajit035**
## Describe your changes
I have changed **EDIT** and **DELETE** button's position

## Screenshots - 
image

## Checklist before requesting a review

- [x] I have performed a self-review of my code.

- [x] Followed the repository's [Contributing Guidelines](https://github.com/dvstechlabs/Noteslify/blob/main/CONTRIBUTING.md).

- [x] I ran the app and tested it locally to verify that it works as expected.

- [x] I have starred the repository.

### CODE REVIEW

File: frontend/src/components/Account/Account.css [hunk 1]
```diff
@@ -44,8 +44,9 @@ header,
 }
 
 .note .details {
-    max-height: 165px;
+    /* max-height: 165px; */
     overflow-y: auto;
+    height: 82%;
```
Comment (biswajit035) [inline frontend/src/components/Account/Account.css:49]: changed the structure of notes card

File: frontend/src/components/LandingPage/LandingPage.css [hunk 2]
```diff
@@ -218,8 +218,7 @@ button:hover {
 }
 
 button,
-a,
-li {
+a {
```
Comment (biswajit035) [inline frontend/src/components/LandingPage/LandingPage.css:221]: changed for better experience

File: frontend/src/components/Notes/Notes.css [hunk 3]
```diff
@@ -148,7 +147,13 @@ header,
     border-radius: 0;
     justify-content: flex-start;
 }
-
```
Comment (biswajit035) [inline frontend/src/components/Notes/Notes.css:151]: changed for better experience

File: frontend/src/components/Notes/Notes.css [hunk 4]
```diff
@@ -44,7 +44,6 @@ header,
 }
 
 .note .details {
-    max-height: 165px;
```
Comment (biswajit035) [inline frontend/src/components/Notes/Notes.css:47]: change for layout

File: frontend/src/components/Notes/Notes.css [hunk 5]
```diff
@@ -283,4 +288,20 @@ form button {
     .bottom-content .settings i {
         font-size: 17px;
     }
+}
```
Comment (biswajit035) [inline frontend/src/components/Notes/Notes.css:291]: added **EDIT** and **DELETE** button here

**Review by biswajit035 — COMMENTED** [review]
Chages done succesfully

**Review by devarshishimpi — APPROVED** [review]

**Comment by devarshishimpi** [comment]
Thanks For Contributing!!

**Pull request merged**